ios/ble/Utility/TransferHandler.swift

```diff
@@ -89,12 +92,19 @@ class TransferHandler {
         if (r.type == .SUCCESS) {
             currentState = States.TransferVerified
             EventEmitter.sharedInstance.emitNearbyMessage(event: "send-vc:response", data: "\"RECEIVED\"")
+            retryCount = 0
             os_log(.info, "Emitting send-vc:response RECEIVED message")
         } else if r.type == .MISSING_CHUNKS {
             currentState = .PartiallyTransferred
-            sendRetryRespChunk(missingChunks: r.missingSequences!)
+            retryCount+=1
```

Comment:
Incrementing here will only allow retry to be `MAXIMUM_RETRY_FRAME_LIMIT -1` times

Reply:
fixing it